roles/inspec/templates/etc/inspec/host-controls/attributes.yml

```diff
@@ -0,0 +1,2 @@
+# BEGIN ANSIBLE MANAGED BLOCK horizon
```

Comment:
I don't believe you need to prefill these, the block in file should create them.